src/goto-checker/single_loop_incremental_symex_checker.cpp

```diff
@@ -168,25 +168,33 @@ operator()(propertiest &properties)
 
 goto_tracet single_loop_incremental_symex_checkert::build_full_trace() const
 {
-  goto_tracet goto_trace;
-  build_goto_trace(
-    equation,
-    equation.SSA_steps.end(),
-    property_decider.get_decision_procedure(),
-    ns,
-    goto_trace);
-
-  return goto_trace;
+  UNREACHABLE;
```

Comment:
🚫 Write a test for it if you want to cover it.

Reply:
Done.